README.md

```diff
@@ -7,37 +7,37 @@
 Package your application, client tools, configuration, and deployment logic into an installer that you can distribute and run with a single command.
 Based on the Cloud Native Application Bundle Specification, [CNAB](https://deislabs.io/cnab), Porter provides a declarative authoring experience that lets you focus on what you know best: your application.
 
-<p align="center">Learn all about Porter at <a href="https://porter.sh">porter.sh</a></p>
+<p align="center">Learn all about Porter at <a href="https://getporter.org">porter.sh</a></p>
```

Comment:
Do we also want to change the displayed text from porter.sh to getporter.org?

Reply:
oops!